src/main-process/proxy/test-actions/analyze.ts

```diff
@@ -60,6 +61,7 @@ const pollTransformStatus = async (token: string): Promise<void> => {
         const requests = details.conf.requests;
         const entries = getEntries();
         markIrrelevantRequests(entries, requests);
+        assignRequestDescriptions(entries, requests);
```

Comment:
maybe this and `markIrrelevantRequests` can be in the same function that process/prepare entries? This way we don't iterate twice on the entries and also on future operations we won't have to add another function here.